package-lock.json

```diff
@@ -6110,7 +6110,11 @@
           "integrity": "sha512-Z5BTk6ZRe4tXXQlkqftmsAUANpXmuwlsF5Oov8ThoMbQRzdGTA1ngYRW160GexgOgjsFOKJz0LYhoNi+2AMBUw==",
           "dev": true,
           "requires": {
+<<<<<<< HEAD
```

Comment:
looks like `package-lock.json` has some unmerged changes that are causing the netlify deploy preview to fail.

This should be fixable by doing the following:
```
rm package-lock.json
npm install
git add package-lock.json
git commit -S 
```

Reply:
complete. no issues.